Updates exec.d on Windows (#203)
buildpacks/spec: buildpack.md

| Input             | Description
|-------------------|----------------------------------------------
| `$0`              | Absolute path of the executable

> | [†](README.md#linux-only)FD 3              | A third open file descriptor
> | [‡](README.md#windows-only)`CNB_EXEC_D_HANDLE`            | Hex representation of an open file handle

Reviewer: ```suggestion
| [†](README.md#linux-only)FD 3              | A third open file descriptor
| [‡](README.md#windows-only) `<handle>`  | An additional open handle
| [‡](README.md#windows-only)`CNB_EXEC_D_HANDLE` | Hexidecimal number for `<handle>`
```
Couldn't grab the line above in the suggestion (to fix spaces) so spacing will need to be fixed if you accept suggestion.

| Output             | Description
|--------------------|----------------------------------------------
| [exit status]      | Pass (0) or error (1+)
| Standard output    | Logs (info)
| Standard error     | Logs (warnings, errors)

> | [†](README.md#linux-only)FD 3 or [‡](README.md#windows-only)`CNB_EXEC_D_HANDLE` | Launch time environment variables (see [Exec.d Output](#execd-output-toml))

Reviewer: ```suggestion
| [†](README.md#linux-only)FD 3 or [‡](README.md#windows-only)`<handle>` | Launch time environment variables (see [Exec.d Output](#execd-output-toml))
```
